tests/functional/e/enum_self_defined_member_5138.py

```diff
@@ -0,0 +1,45 @@
+# pylint: disable=missing-docstring
```

Comment:
```suggestion
"""Tests for https://github.com/PyCQA/pylint/pull/5847."""

# pylint: disable=missing-docstring
```

Reply:
I think this is superfluous because the file name contains the issue number and the purpose of the tests, but I'll shove some English in a docsttring

Reply:
Somewhat agree, but having a direct link is always easier to revisit an issue imo. Especially considering the age of the project. There are multiple references to "issue xx" which don't correspond to a Github issue and then be issues on `gitlab` or any of the previous hosting servers of the project. That's why I like explicit links if possible.